README.md

```diff
@@ -247,8 +247,8 @@ A maximum of 10000 ids may be supplied.
 
 ## Requirements
 
-* Python 3.6+
-* MongoDB 2.6+
+* Python 3.9.19
```

Comment:
```suggestion
* Python 3.9+
```
I don't think we have to be that specific. Actually, now I think of it, we should change the Pipefile to just be 3.9

Reply:
I think we should give the version that works and well tested. In GHA and docker is 3.9.19 though.

Reply:
But that means just to test the service, the user needs to install exactly 3.9.19, not 3.9.18 or any other version they might have. We still specify the version in the Dockerfile and GHA, like you say, so we can be exact there.

Reply:
:+1: